litellm/llms/azure/responses/transformation.py

```diff
@@ -18,16 +24,73 @@
 else:
     LiteLLMLoggingObj = Any
 
-
 class AzureOpenAIResponsesAPIConfig(OpenAIResponsesAPIConfig):
     def validate_environment(
         self,
         headers: dict,
         model: str,
+        litellm_params: dict,
         api_key: Optional[str] = None,
     ) -> dict:
+        azure_ad_token_provider = litellm_params.get("azure_ad_token_provider")
```

Comment:
can we make this new block of logic a simple helper in azure/common_utils.py ? 

Then we can re-use for new endpoints like /image/edits etc

Reply:
I mean the entire new section of code you added, that should be a simple helper util